cmd/create.go

```diff
@@ -191,3 +194,14 @@ func normalizeSubnetworkID(options *options.Options) *string {
 	// {{name}}
 	return ptr.Ptr(fmt.Sprintf("projects/%s/regions/%s/subnetworks/%s", project, region, sn))
 }
+
+func getMaintenancePolicy(machineType string) string {
+	gpuInstancePattern := `^[agn][0-9]`
+	re := regexp.MustCompile(gpuInstancePattern)
```

Comment:
Wouldn't it make sense to move this to a private global variable?

Given that must compile might fail here on invocation of the function and not on program start up.

Reply:
Done :+1: